ui/admin/app/templates/scopes/scope/onboarding/quick-setup/create-resources/index.hbs

```diff
@@ -1,3 +1,15 @@
-<button type='button' {{on 'click' (route-action 'submit')}}>{{t
-    'actions.submit'
-  }}</button>
\ No newline at end of file
+{{page-title 'CreateResources'}}
```

Comment:
Do we want this page title? If so, I will need to add them to mine and we should probably change it to "Create Resources".

Reply:
we have them in the templates, and we should switch to follow this pattern `{{page-title (t 'titles.new')}}`